.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,33 @@
+name: Continuous integration
+on:
+  pull_request:
+  workflow_call:
+  workflow_dispatch:
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-version: ['7.4', '8.0', '8.1', '8.2', '8.3']
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+    - name: Use PHP
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: ${{ matrix.php-version }}
+    - name: Cache the vendor dir
+      uses: actions/cache@v4
+      with:
+        path: vendor
+        key: ${{ runner.os }}-${{ matrix.php-version }}-vendor-${{ hashFiles('composer.lock') }}
+    - name: Install
+      run: composer install --no-interaction --prefer-dist
+    - name: Lint
+      run: composer lint
+#    - name: Test
+#      run: composer test
+#    - name: Upload coverage report
+#      uses: codecov/codecov-action@v4
+#      with:
+#        files: ./coverage/coverage-final.json
```

Comment:
Is there a reason this is left in?

Reply:
To force me to add tests 😂 

We can add it later, there's no real benefit on having this block.